jobs/README.md

```diff
@@ -4,6 +4,25 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Perl/PHP developer @ Pancreta Bank
+
+Pancretan Co-op Bank wishes to hire a full developer to be responsible for building and maintaining Perl and PHP scripts and applications.
```

Comment:
Is `full developer` about full-time?

Reply:
Yes, sorry...should be: "Pancretan Co-op Bank wishes to hire a full time developer ...."